Packs/JamfProtect/Integrations/JamfProtectEventCollector/JamfProtectEventCollector_test.py

```diff
@@ -118,14 +120,33 @@ def test_calculate_fetch_dates_without_arguments(client):
     assert end_date == MOCK_TIME_UTC_NOW
 
 
-@pytest.mark.parametrize("with_alert_next_page, with_audit_next_page",
-                         [(False, False), (True, False), (False, True), (True, True)],
-                         ids=["Case 1: No next pages for alerts and audits",
-                              "Case 2: Next page for alerts, no next page for audits",
-                              "Case 3: No next page for alerts, next page for audits",
-                              "Case 4: Next pages for both alerts and audits"]
-                         )
-def test_nextTrigger(with_alert_next_page, with_audit_next_page, mocker):
+@pytest.mark.parametrize(
```

Comment:
Maybe it'll be easier to triple-`parametrize` instead of writing all 8 boolean permutations (P.S we have 7/8, is one missing, or is it not that important?). 

The `id`s are quite explanatory and not necessarily needed, one checking the test will see something like `True,False,True` and will be able to understand (by the argument names) what it means quite easily. Your call.